src/app/pages/admin-page/admin-page.module.ts

```diff
@@ -0,0 +1,19 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { RouterModule } from '@angular/router';
+
+import { components, routes } from './admin-page-routing.module';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    RouterModule.forChild(routes)
+  ],
+  declarations: [
```

Comment:
there are no need to use array destruction into new array, just use components arr as obj value